sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -629,7 +628,12 @@ public Read withTemplateCompatibility() {
        */
       EXPORT,
 
-      /** Read the contents of a table directly using the BigQuery storage API. */
+      /**
+       * Read the contents of a table directly using the BigQuery Storage API.
+       *
```

Comment:
How about: "The BigQuery Storage API is currently in beta and support for storage API in Beam is experimental."